docs/reference-guides/interactivity-api/README.md

```diff
@@ -40,15 +40,30 @@ You can start creating interactions once you set up a block development environm
 
 To indicate that the block [supports](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-supports/) the Interactivity API features, add `"interactivity": true` to the `supports` attribute of the block's `block.json` file.
 
-```
+```json
+// block.json
 "supports": {
     "interactivity": true
 },
 ```
 
+Refer to the [`interactivity` support property docs](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-supports/#interactivity) to get a more detailed description of this property.
+
+#### Load Interactivity API Javascript code with `viewScriptModule`
+
+The Interactivity API script requires using Javascript modules that can be [natively loaded in WordPress from v6.5](https://make.wordpress.org/core/2024/03/04/script-modules-in-6-5/). The JavaScript files using the Interactivity API, should be enqueued via the [`viewScriptModule` block metadata](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#view-script-module):
```

Comment:
The "WordPress v6.5" looks inconsistent, other places in this document we refer to "WordPress 6.5". I left some other suggestions here as well trying to make this clear and precise, please do with it what you see fit:

```suggestion
The Interactivity API provides the `@wordpress/interactivity` Script Module. JavaScript using the Interactivity API should be implemented as Script Modules so they can depend on `@wordpress/interactivity`. [Script Modules have been available since WordPress from 6.5](https://make.wordpress.org/core/2024/03/04/script-modules-in-6-5/). Block can use [`viewScriptModule` block metadata](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#view-script-module) to easily enqueue their Script Modules:
```